keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -991,13 +1007,26 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+Rollouts and rollbacks can fail in many ways. During the first rollout of the
+feature, there will be a mixed state of control planes using and not using
+coordinated leader election. Components not using CLE will race to obtain the
+best leader while the ones using CLE will defer the CLE controller to assign
+themselves as leader.
```

Comment:
But only one leader can ever be elected, correct?  Maybe clarify here.  The way this is phrased it's unclear what level of risk there is, but my understanding is that, while we don't guarantee _coordinated_ leader election, we still get leader election.

Reply:
Updated